src/main/scala/com/atomist/rug/test/gherkin/DefaultExecutableFeatureFactory.scala

```diff
@@ -17,9 +19,13 @@ object DefaultExecutableFeatureFactory extends ExecutableFeatureFactory {
                                     definitions: Definitions,
                                     rugAs: ArtifactSource,
                                     rugs: Option[Rugs],
-                                    listeners: Seq[GherkinExecutionListener]): AbstractExecutableFeature[_,_] = {
+                                    listeners: Seq[GherkinExecutionListener]): AbstractExecutableFeature[_] = {
     if (f.definition.path.contains("project"))
       new ProjectManipulationFeature(f, definitions, rugAs, rugs, listeners)
+    else if (f.definition.path.contains("handlers/command"))
+      new CommandHandlerFeature(f, definitions, rugAs, rugs, listeners)
+    else if (f.definition.path.contains("handlers/event"))
```

Comment:
I've cleaned this up a bit. We need to update `AtomistConfig` test directory, deciding whether its `tests` or `test`. It was originally `tests` but `test` seems more common.